src/django_components/component.py

```diff
@@ -50,6 +41,16 @@
 from django_components.template_parser import process_aggregate_kwargs
 from django_components.utils import gen_id
 
+# TODO_DEPRECATE_V1 - REMOVE IN V1, users should use top-level import instead
```

Comment:
To avoid needless breaking changes, files from component registry are re-exported in `component.py` until v1.